### DISCUSSION TITLE
"Shorten The URL" button not working

### DISCUSSION THREAD

**Discussion by ickyshuffle** [YOURLS/YOURLS#3266] [General]
I *feel* like this should be simple but I cannot find any logs to help and could not find any references to this problem with Github or Google searches.  When I try to create a short URL, the button does not take any action.

### **Technical details**

YOURLS v1.8.2
PHP version:  7.4.19

### **Reproducible Bug Summary**

1. Log into my yourls admin screen
2. Click "Admin interface"
3. Enter webpage and desired short url
5. Click "Shorten The URL" button
6. Nothing happens

### **Troubleshooting**

- PHP is working (I can get to the site and navigate across the different links within the YOURLS interface
- MySQL (MariaDB) is working because the YOURLS DB is populated with the 3 sample URLs.
- I get 200/OK's from the pages themselves.

**Comment by ozh** [comment]
Most likely here : https://github.com/YOURLS/YOURLS/wiki#support--troubleshooting
